apps/term-apply/pkg/transfer/copy_from_client.go

```diff
@@ -43,8 +44,9 @@ func (c *copyFromClientHandler) Mkdir(s ssh.Session, entry *scp.DirEntry) error
 }
 
 func (c *copyFromClientHandler) Write(s ssh.Session, entry *scp.FileEntry) (int64, error) {
-	fin := s.User()
```

Comment:
I was like whats `fin` mean, but I think that was me initially making up vars while hacking on something :laughing:

Reply:
```
    There are only two hard things in Computer Science: cache invalidation and naming things.

    -- Phil Karlton
```